assets/scripts/src/choices.js

```diff
@@ -647,8 +647,6 @@ class Choices {
 
     if (shouldFlip) {
       this.containerOuter.classList.add(this.config.classNames.flippedState);
-    } else {
-      this.containerOuter.classList.remove(this.config.classNames.flippedState);
```

Comment:
I was adding `is-flipped` class dynamically after choices init in certain scenarios of mine. And then `showDropdown` method was removing this class, because there was enough space.

But I can remove it, if you want and do it with different class on my own.